core/vm/evm.go

```diff
@@ -175,6 +176,13 @@ func (evm *EVM) SetBlockContext(blockCtx BlockContext) {
 // the necessary steps to create accounts and reverses the state in case of an
 // execution error or failed value transfer.
 func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas uint64, value *big.Int) (ret []byte, leftOverGas uint64, err error) {
+	if v, ok := evm.Config.Tracer.(EVMLoggerExtended); evm.Config.Tracer != nil && ok {
```

Comment:
Please leave out these for now.

Reply:
FYI, I have locally the change that moves up existing `CaptureStart/Enter`, just let me know if you need it.